bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/ThingHandlerCallback.java

```diff
@@ -127,6 +131,17 @@
      */
     ChannelBuilder editChannel(Thing thing, ChannelUID channelUID);
 
+    /**
+     * Creates a list of {@link ChannelBuilder}s which are preconfigured with values from the given channel group type.
+     *
+     * @param channelGroupUID the UID of the channel group to be created
+     * @param channelGroupTypeUID the channel group type UID for which the channel should be created
+     * @return a list of preconfigured ChannelBuilders
+     * @throw {@link IllegalArgumentException} if the referenced channel group type is not known
```

Comment:
Hm, I never seen this before... Isn't the "word" after `@throw` always the excpetion class and so already a "link" created by the JavaDoc tooling?

Reply:
Nice catch. I copy & pasted it. Follow up question: Do you know the `@throw` tag anyway? Correct syntax should be `@throws`. It is used in the wrong way throughout for all three methods.

Reply:
Nice catch, too :wink: